### ISSUE TITLE
Docs: explicitly mention that omitting `npm` version means using version bundled with Node? (PR#957)

### ISSUE THREAD

**Issue by bobo96run** [volta-cli/volta#1682]
Re-hi and thank you again for this awesome tool!

I am a new user of Volta, and when starting to configure my projects, I naively explicitly specified `npm` version alongside `node` in `package.json`, e.g.:
```json
{
  "volta": {
    "node": "14.21.3",
    "npm": "6.14.18"
  }
}
```
But now I realize that I can omit the `npm` spec, and Volta will automatically use the version that is bundled with the specified Node version (which is in this case indeed 6.14.18), as introduced by PR #957 (see issue #953).

I think it would be worthwhile mentioning this handy behaviour explicitly in the documentation?
Maybe here: https://docs.volta.sh/guide/understanding#pinning-node-engines

**Comment by bobo96run** [comment]
Probably related to https://github.com/volta-cli/volta/issues/1542?

**Comment by lognaturel** [comment]
I agree this would be very helpful, would a contribution be welcome?

I'm on the same team as @alxndrsn who filed https://github.com/volta-cli/volta/issues/2085 and we just once again had to refresh our memory on this default behavior.
